File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/fragment/NotificationViewPositionFragment.java
package com.regula.facesample.items.customization.fragment;

import android.graphics.RectF;
import android.view.View;

import androidx.constraintlayout.widget.Guideline;

import com.regula.facesample.R;
import com.regula.facesdk.fragment.FaceDefaultUiFragment;

/**
 * Created by Sergey Yakimchik on 6.09.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class NotificationViewPositionFragment extends FaceDefaultUiFragment {

    @Override
    public void onFaceAreaViewPositionUpdated(RectF rect) {
        View view = getView();
        if (view == null)
            return;

        int topMargin = (int) rect.top;
        topMargin += rect.height() * 0.4;

        Guideline notificationGuideline = view.findViewById(R.id.topNotificationGuideline);
        notificationGuideline.setGuidelinePercent(-1);
        notificationGuideline.setGuidelineBegin(topMargin);
    }
}

File: BasicSample/app/src/main/java/com/regula/facesamplekotlin/items/fragments/NotificationViewPositionFragment.kt
package com.regula.facesamplekotlin.items.fragments

import android.graphics.RectF
import androidx.constraintlayout.widget.Guideline
import com.regula.facesamplekotlin.R
import com.regula.facesdk.fragment.FaceDefaultUiFragment

class NotificationViewPositionFragment : FaceDefaultUiFragment() {

    override fun onFaceAreaViewPositionUpdated(rect: RectF) {
        val view = view ?: return
        var topMargin = rect.top.toInt()
        topMargin += (rect.height() * 0.4).toInt()
        val notificationGuideline: Guideline = view.findViewById(R.id.topNotificationGuideline)
        notificationGuideline.setGuidelinePercent(-1f)
        notificationGuideline.setGuidelineBegin(topMargin)
    }
}